src/web/lib/components/ThemeColorsEditor/index.scss

```diff
@@ -20,8 +20,9 @@
   padding: $grid * 4;
   position: relative;
   flex: 1 1 640px;
-  height: 320px;
+  height: 350px;
   overflow: auto;
+  padding-bottom: 10px;
```

Comment:
It hould be `margin-bottom`. `padding-bottom` is inside the box, which would still be hidden by the scrollbars.

margin-bottom is outside the box.